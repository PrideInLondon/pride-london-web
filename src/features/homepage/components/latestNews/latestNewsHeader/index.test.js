import React from 'react'
import { shallow } from 'enzyme'
import LatestNewsHeader from '.'

describe(LatestNewsHeader.name, () => {
  it('renders with default values', () => {
    const wrapper = shallow(<LatestNewsHeader />)
    expect(wrapper).toMatchSnapshot()
  })
})
