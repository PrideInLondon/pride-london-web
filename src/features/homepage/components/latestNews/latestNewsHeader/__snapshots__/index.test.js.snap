// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LatestNewsHeader renders with default values 1`] = `
<styles__Header>
  <styles__Title>
    Latest News
  </styles__Title>
  <styles__HiddeOnlyMobile>
    <CTALink
      to="/news-and-views"
    >
      View all news
    </CTALink>
  </styles__HiddeOnlyMobile>
</styles__Header>
`;
